ui/component/or-mwc-components/src/or-mwc-menu.ts

```diff
@@ -51,20 +52,52 @@ declare global {
     }
 }
 
-export function getContentWithMenuTemplate(content: TemplateResult, menuItems: (ListItem | ListItem[] | null)[], selectedValues: string[] | string | undefined, valueChangedCallback: (values: string[] | string) => void, closedCallback?: () => void, multiSelect = false, translateValues = true, midHeight = false, fullWidth = false): TemplateResult {
+export function positionMenuAtElement<T extends OrMwcMenu>(
+    menu: T,
+    hostElement?: HTMLElement
+): T {
+    if (!hostElement) {
+        hostElement = document.body;
+    }
+    const rect = hostElement.getBoundingClientRect();
+
+    // Applying a style that is calculated from the runtime coordinates of
+    // the host element.
+    Object.assign(menu.style, {
+        position: 'fixed',
+        top: `${rect.bottom}px`,
+        left: `${rect.left}px`,
+        zIndex: '1000',
+        display: 'block'
+    });
+
+    return menu;
+}
+
+export function getContentWithMenuTemplate(content: TemplateResult, menuItems: (ListItem | ListItem[] | null)[], selectedValues: string[] | string | undefined, valueChangedCallback: (values: string[] | string) => void, closedCallback?: () => void, multiSelect = false, translateValues = true, midHeight = false, fullWidth = false, menuId = "menu", fixedToHost = false): TemplateResult {
+    let menuRef: OrMwcMenu | null = null;   // Reference to the menu
 
     const openMenu = (evt: Event) => {
         if (!menuItems) {
             return;
         }
 
+        if (fixedToHost && menuRef) {
```

Comment:
Can you maybe explain in what cases `fixedToHost` needs to be used?
I know when it's set to `true`, it will use `position: fixed`, allowing overflow outside the container.
But there is probably a reason why it is `false` by default.

So a small code comment for explanation would be nice here.

Reply:
I noticed that, so good that you did haha

No I was just curious on **when/where** it should be used.
As in, "in what scenarios should this value be set to true?"

But the parameter is self-explanatory for me, so it's OK